docs/uml-class-diagrams/admin/test/SatelliteTests/SatelliteTests.puml

```diff
@@ -2,7 +2,14 @@
 
 package edu.oakland.test.admin {
     class SatelliteTests {
-        
+       + void satNameInIsSatNameOut()
+       + void strengthInIsStrengthOut()
+       + void locInIsLocOut()
+       + void setStrengthIsStrength()
+       + void newLatPointUsedIncrementLat()
+       + void newLngPointUsedIncrementLng()
+       + void newTimeIsIncrementedByHour()
```

Comment:
These internal methods will not be tested directly, but will be covered in a test case like getLocation0(), getLocation1(), getLocationN() where you physically call the Satellite.getLocation() method n times, then check to make sure that the last one is incremented as much as it needs to be for that particular point. I've got an example on how to code this coming out in the near-ish future.